app/api/access_codes.py

```diff
@@ -93,12 +94,12 @@ def query(self, view_kwargs):
         if view_kwargs.get('user_id'):
             user = safe_query_kwargs(User, view_kwargs, 'user_id')
             if not has_access('is_user_itself', user_id=user.id):
-                raise ForbiddenError({'source': ''}, 'Access Forbidden')
+                raise ForbiddenError({'parameter': 'user_id'}, 'Access Forbidden')
             query_ = query_.join(User).filter(User.id == user.id)
         if view_kwargs.get('ticket_id'):
             ticket = safe_query_kwargs(Ticket, view_kwargs, 'ticket_id')
             if not has_access('is_coorganizer', event_id=ticket.event_id):
-                raise ForbiddenError({'source': ''}, 'Access Forbidden')
+                raise ForbiddenError({'parameter': 'event_id'}, 'Access Forbidden')
```

Comment:
In this situation what will be source or pointer? ticket_id or event_id or user_id..!!

Reply:
ticket_id, paramter